platforms/ide/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeCppApplicationProjectIntegrationTest.groovy

```diff
@@ -43,7 +44,7 @@ class XcodeCppApplicationProjectIntegrationTest extends AbstractXcodeCppProjectI
         return new CppApp()
     }
 
-    @Requires(UnitTestPreconditions.HasXCode)
+    @Requires(value = [UnitTestPreconditions.HasXCode, IntegTestPreconditions.NotEmbeddedExecutor], reason = "Need a Gradle install to pass to xcodebuild")
```

Comment:
💅 Using `import static` as in other classes

Reply:
Fixed.